csbindgen/src/type_meta.rs

```diff
@@ -463,20 +459,16 @@ pub fn build_method_delegate_if_required(
     method_name: &String,
     parameter_name: &String,
 ) -> Option<String> {
-    let emit_from_struct = false;
-
     match &me.type_kind {
         TypeKind::Function(parameters, return_type) => {
-            if emit_from_struct && !options.csharp_use_function_pointer {
```

Comment:
So... if `emit_from_struct` is always false, can this actually ever be the case? I removed the branch; this was likely a copy/paste leftover, but might be worth checking if it isn't some kind of bug.